activities/codebase-stewardship/success-for-a-codebase.md

```diff
@@ -0,0 +1,11 @@
+---
+type: resource
+---
+
+# Success for a codebase
+
+This is how we define the sucess of codebase stewardship on a codebase:
+
+> Success = (contributorsx2)x(maintainersx10)
+
+Note: 'users' has not been included as this is dificult to measure 
```

Comment:
I'd remove the 'Note:' as this is fine inline information and is not extra important in usage (or problematic if not taken into account).